### ISSUE TITLE
Fix image sizings

### ISSUE THREAD

**Issue by fnocetti** [huridocs/uwazi-documentation#3]
Markdown provides no way to set the images size so we need to use image files with the size we need them to be shown on the document.
By default it will use the maximum size it can fit on the article. This is particularly important for inline images (like those showing app icons) as they will use all the space they can, breaking the text.

**Comment by txau** [comment]
@fnocetti I guess regular html is supported for images in GitHub since a while ago:
```
<img src="https://github.com/favicon.ico" width="48">
```
Resulting in:
<img src="https://github.com/favicon.ico" width="48">

**Comment by fnocetti** [comment]
@txau you're right. But I forgot to say that using html inside the markdown messes with the PDF generation.

**Comment by nwyu** [comment]
All icons have been resized. Still need to resize other images and scrub files for `[image alt text]` so this text does not appear improperly on the pdf.
